roles/tm-deepsec-aws-setup/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+# tasks file for tm_silent_setup
+
+- name: Ensure bash, OpenSSl, and libssl are the latest versions
+  yum: name={{ item }} update_cache=true state=latest
```

Comment:
The `name` key of `yum` module can accept a lst of packages. This is a better approach than the use of `with_items`. Also, can you avoid inline parameters (e.g: `state=latest`). This is also true for the rest of the file.

Reply:
+1